tests/files/das-root.json

```diff
@@ -1,70 +1,68 @@
 {
```

Comment:
The change in this file is small, but because the indentation was 4 and I made it to 2 it seem as my file is a totally new one.